src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -86,6 +107,16 @@
         expandable
         :label="$t('transaction.advancedDetails')"
       >
+        <DetailsItem
+          v-if="decodedCallData && decodedCallData.functionName"
+          :label="$t('modals.confirmTransactionSign.functionName')"
+          :value="decodedCallData.functionName"
```

Comment:
Function name is displayed here for the 2nd time.

Reply:
Yes, that was intended.